resolwe_bio/processes/support_processors/igv_preprocess.yml

```diff
@@ -0,0 +1,53 @@
+# ============
+# IGV preprocess
+# ============
+
+---
+
+- slug: igv-preprocess-gtf
+  name: Sort and index GTF files
+  requirements:
+    expression-engine: jinja
+    executor:
+      docker:
+        image: resolwebio/utils
+  data_name: "Preprocessed GTF file for IGV"
+  version: 0.0.1
+  type: data:annotation:gtf
+  category: analyses
+  description: >
+    Sort and index largeGTF files with igvtools tool.
+  input:
+    - name: annotation
+      label: Annotation (GTF)
+      type: data:annotation:gtf
+      description: >
+        Annotation in GTF format.
+      validate_regex: '\.(gtf)(|\.gz|\.bz2|\.tgz|\.tar\.gz|\.tar\.bz2|\.zip|\.rar|\.7z)$'
+  output:
+    - name: gtf
```

Comment:
Rename to `annot`, as defined in [abstract process](https://github.com/genialis/resolwe-bio/blob/master/resolwe_bio/processes/annotation/abstract.yml)